app/src/main/res/layout/activity_settings.xml

```diff
@@ -25,23 +25,25 @@
                 android:id="@+id/profilePictureView"
                 android:layout_width="@dimen/large_profile_picture_size"
                 android:layout_height="@dimen/large_profile_picture_size"
-                android:layout_marginTop="@dimen/medium_spacing" />
+                android:layout_marginTop="@dimen/medium_spacing"
+                android:contentDescription="@string/AccessibilityId_profile_picture" />
 
             <RelativeLayout
                 android:id="@+id/ctnGroupNameSection"
+                android:contentDescription="@string/AccessibilityId_username"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginLeft="@dimen/large_spacing"
                 android:layout_marginTop="@dimen/small_spacing"
                 android:layout_marginRight="@dimen/large_spacing">
 
                 <EditText
-                    style="@style/SessionEditText"
                     android:id="@+id/displayNameEditText"
+                    style="@style/SessionEditText"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_centerInParent="true"
-                    android:textAlignment="center"
```

Comment:
Is this `textAlignment` change intentional?

Reply:
Nope, if it was it was done by @hjubb maybe?

Reply:
quite possibly, it might have been that because we had `layout_width="wrap_content"` there was no reason to center the text alignment here and I removed it at the same time as adding that content description

Reply:
Will leave it out then